mod-orders-storage/schemas/location.json

```diff
@@ -24,6 +24,11 @@
     "quantityPhysical": {
       "description": "quantity of physical items",
       "type": "integer"
+    },
+    "tenantId": {
+      "description": "UUID of the associated tenant",
```

Comment:
think that better to add into description, that property is applicable only for ECS based envs, and the same also for other new tenantId params

Reply:
Added